src/main/java/com/ericsson/ei/jmespath/JmesPathInterface.java

```diff
@@ -64,6 +64,9 @@ public JsonNode runRuleOnEvent(String rule, String event) {
             JsonNode eventJson = objectMapper.readValue(event, JsonNode.class);
             result = expression.search(eventJson);
         } catch (Exception e) {
+        	log.info("runRuleOnEvent failed for given arguments:");
```

Comment:
This isn't thread safe right? I think it's good that these messages comes together all the time. Should they maybe be put inside the same info and then format it to your needs?

Reply:
Maybe the log output should be error and not info.

Reply:
I agree, put in one log warning. log.warning(MSG1 + "\n" + MSG2 + "\n" + MSG3).